Update Linux build instructions (#1965)
maplibre/maplibre-native: platform/linux/README.md

# Linux

> This guide explains how to get started building and running MapLibre Native on Linux. The guide focusses on a Ubuntu 22.04 clean docker image, but should be adaptible to other distributions. The build process should give you a set of `.a` files that you can use to include MapLibre Native in other C++ projects, as well as a set of executables that you can run to render map tile images and test the project.

Reviewer: Let's remove the comment that it is adaptible to other distributions, because the version of the compiler is not specified.

```suggestion
This guide explains how to get started building and running MapLibre Native on Linux. The guide focusses on a Ubuntu 22.04. The build process should give you a set of `.a` files that you can use to include MapLibre Native in other C++ projects, as well as a set of executables that you can run to render map tile images and test the project.
```

## Requirements

The following system libraries need to be installed.

```bash

> apt install libcurl4-openssl-dev libglfw3-dev libuv1-dev libpng-dev libicu-dev libjpeg8-dev libwebp-dev xvfb

Reviewer: ```suggestion
apt install libcurl4-openssl-dev libglfw3-dev libuv1-dev libpng-dev libicu-dev libjpeg-turbo8-dev libwebp-dev xvfb
```

> apt install libcurl4-openssl-dev libglfw3-dev libuv1-dev libpng-dev libicu-dev libjpeg8-dev libwebp-dev xvfb

Reviewer: libjpeg is less efficient than libjpeg-turbo as far as I know.

Author: I think the turbo was not found...

Reviewer: It should exist https://packages.ubuntu.com/search?searchon=sourcenames&keywords=libjpeg-turbo

```

Optional: `libsqlite3-dev` (also available as vendored dependency).

The following tools need to be available.

```bash
apt install clang git cmake ccache ninja-build pkg-config
```

## Docker

> Alternatively to the above requirements, you can use a docker container to build MapLibre Native:

Reviewer: ```suggestion
You can use a Docker container to build MapLibre Native. A `Dockerfile` that installes the required dependencies when the image is built is provided in this directory.
```

> Create image with:

Reviewer: ```suggestion
Build image with:
```
